src/ui/qgsprojectpropertiesbase.ui

```diff
@@ -1737,7 +1737,7 @@
                     <string/>
                    </property>
                    <property name="currentIndex">
-                    <number>0</number>
+                    <number>2</number>
```

Comment:
This one should stay at 0 to always open on the "services capabilities" tab and not WMTS. Right?
```suggestion
                    <number>0</number>
```